app/models/news.rb

```diff
@@ -11,9 +11,29 @@ def is_published?
   end
 
 
-  extend Enumerize
 
-  scope :published, -> { where 'published_at <= ?', DateTime.now}
-  scope :unpublished, -> { where 'published_at > ?', DateTime.now}
+  scope :published, ->   { where('published_at <= ?', DateTime.now).where.not(state: :removed)}
```

Comment:
Line is too long. [95/80]<br>Unnecessary spacing detected.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.